vite.config.ts

```diff
@@ -12,6 +12,11 @@ export default defineConfig({
   plugins: [react(), viteTsconfigPaths(), svgr(), eslint()],
   build: {
     outDir: "build",
+    rollupOptions: {
```

Comment:
Wanna add [the link](https://github.com/vitejs/vite/issues/11804#issuecomment-2009619365) here so we know why we did this?